airflow/providers/google/cloud/hooks/dataproc.py

```diff
@@ -69,7 +69,7 @@ def __init__(
         if properties is not None:
             self.job["job"][job_type]["properties"] = properties
 
-    def add_labels(self, labels: dict) -> None:
+    def add_labels(self, labels: Optional[dict]) -> None:
```

Comment:
If I remember correctly we were using `Optional` to indicate that the parameter is optional and usually has default value of `None` as described in [docs](https://docs.python.org/3/library/typing.html#typing.Optional). So I would suggest:
```suggestion
    def add_labels(self, labels: Optional[dict] = None) -> None:
```
to be consistent, WDYT?

Reply:
Good point.